### DISCUSSION TITLE
Context menu for a `TextInput`?

### DISCUSSION THREAD

**Discussion by BrandonXLF** [slint-ui/slint#6679] [Q&A]
Is there currently a way to add a context menu to a `TextInput`?

My current idea is to have a `TouchArea` above the `TextInput` that only handles right-click events and shows a `PopupWindow`, but that is not possible until the gesture recognition mentioned in #5772 is added.

**Comment by ogoffart** [comment]
Yes. You can have something like this:

```slint
ContextMenuArea {
    TextInput { text: "Abc"; }
    Menu {
        MenuItem {
            title: "Hello";
            activated => {debug("Hello");}
        }
    }
}
```
